app/controllers/registrations_controller.rb

```diff
@@ -5,9 +5,12 @@ def create
       params,
       user: current_user,
       name: I18n.t('registrations.your_first_garden'),
-      description: I18n.t('registrations.we_automatically_generated_this_garden')
+      description: I18n.t('registrations.generated_this_garden'),
+      average_sun: "Full Sun",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.